frontend/src/routes/manage/Video/Shared.tsx

```diff
@@ -161,7 +161,7 @@ const ManageVideoNav: React.FC<ManageVideoNavProps> = ({ event, active }) => {
     const header = (
         <div css={{ display: "flex", flexDirection: "column" }}>
             <Link to={videoLink} css={{
-                dislay: "block",
+                display: "block",
```

Comment:
lol

seems like it works fine without this. Chrome dev tools show me in the current form the `a` element is computed to have `display: block`. No idea :shrug: But ye sure lets leave it like that.